packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -32,6 +32,8 @@ import { formatDate, formatTime } from '../../utils/date';
 import reduxFormFieldAdapter from '../../utils/redux-form-field-adapter';
 
 import './SearchForm.css';
+import { trackEvent } from '../../utils/tracking';
+import { CATEGORY_BASE } from './SearchForm.track';
```

Comment:
Imports are sorted alphabetically with more dots under less dots and css files that are imported for the side effect go below the other imports

	import a from './a';
	import z from './z';
	import b from '../b';

	import 'a.css';

Please adjust this in all cases